packages/amplication-client/src/Resource/constants.ts

```diff
@@ -161,6 +162,7 @@ export function prepareMessageBrokerObject(
     name: "My message broker",
     description: "",
     resourceType: models.EnumResourceType.MessageBroker,
+    codeGenerator: models.EnumCodeGenerator.NodeJs,
```

Comment:
Since `codeGenerator` is optional in `Resource`, and we are in a Message Broker, isn't it better to leave it undefined (not to add this line)?

Reply:
the value on the server side is non-nullable so it will be easier to maintain this feature on the long run, for now and as long as we don't have other generator, the default is anyway "Node.js" 
so, I thought that for now, this should be a more maintainable approach that will prevent drifting from the original design as we add more resources